pages/search-beta.tsx

```diff
@@ -53,15 +53,15 @@ const Search = ({ serverState, isServerRendered, serverUrl }: Props) => (
     <IntlProvider locale="en" defaultLocale="en" messages={englishTexts}>
       <InstantSearch
         searchClient={isServerRendered ? searchClientServer : searchClient}
-        indexName="test_assets"
+        indexName="Data_Set"
         routing={{
           router: history({
             /*  @ts-ignore */
             getLocation: () => (typeof window === "undefined" ? new URL(serverUrl) : window.location),
           }),
         }}
       >
-        <Configure hitsPerPage={50} snippetEllipsisText="..." />
+        <Configure hitsPerPage={50} snippetEllipsisText="..." attributesToSnippet={["excerpt", "description"]} />
```

Comment:
would there be a way, do you think, for us abstract the `attributesToSnippet`? in case the model changes, so we don't have hard coded values

Reply:
Good question 👍  When we have created the search indexing service, the `attributesToSnippet` will be added there. The Configure is just a way to do things until we get there (might be that we need it for hitsPerPage, I don't remember 100%)

[See here in Envis for an example](https://github.com/equinor/energyvision/blob/main/search/IndexSanityContent/topic/algolia.ts)